src/Predict/jobData.js

```diff
@@ -15,23 +15,23 @@ const jobCards = {
   },
   caliban: {
     file: 'tiff_stack_examples/3T3_nuc_example_256.tif',
-    name: 'Cell Tracking',
-    model: 'Caliban segments and tracks objects over time and creates a lineage file for division information.',
-    inputs: 'A single-channel image stack (3D TIFF).',
-    resolution: 'Cell Tracking expects data that is approximately 20X (0.65 μm/pixel)',
+    name: 'Caliban',
+    model: 'Caliban segments and tracks nuclei over time and creates a lineage file with division information.',
+    inputs: 'A single-channel image stack of fluorescent nuclei (3D TIFF). Caliban expects data that is approximately 20X (0.65 μm/pixel).',
+    resolution: 'Caliban expects data that is approximately 20X (0.65 μm/pixel).',
     thumbnail: 'thumbnails/3T3_nuc_example_256.png',
-    scaleEnabled: true,
-    channelEnabled: true,
+    scaleEnabled: false,
```

Comment:
We don't use these variables (`scaleEnabled` and `channelEnabled`) anywhere now that we have individual files for each form, but we could revisit deprecating them some other time

Reply:
I'll open an issue for a global update to address those variables.